cdap-hbase-compat-0.96/src/main/java/co/cask/cdap/data2/util/hbase/DefaultHBase96DDLExecutor.java

```diff
@@ -16,98 +16,22 @@
 
 package co.cask.cdap.data2.util.hbase;
 
-import co.cask.cdap.spi.hbase.ColumnFamilyDescriptor;
-import co.cask.cdap.spi.hbase.CoprocessorDescriptor;
 import co.cask.cdap.spi.hbase.HBaseDDLExecutor;
 import co.cask.cdap.spi.hbase.TableDescriptor;
-import org.apache.hadoop.conf.Configuration;
-import org.apache.hadoop.fs.Path;
-import org.apache.hadoop.hbase.HColumnDescriptor;
 import org.apache.hadoop.hbase.HTableDescriptor;
-import org.apache.hadoop.hbase.TableName;
-import org.apache.hadoop.hbase.io.ImmutableBytesWritable;
-import org.apache.hadoop.hbase.io.compress.Compression;
-
-import java.io.IOException;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
 
 /**
  * Implementation of the {@link HBaseDDLExecutor} for HBase 0.96
  */
 public class DefaultHBase96DDLExecutor extends DefaultHBaseDDLExecutor {
 
-  private HColumnDescriptor getHColumnDesciptor(ColumnFamilyDescriptor descriptor) {
-    HColumnDescriptor hFamily = new HColumnDescriptor(descriptor.getName());
-    hFamily.setMaxVersions(descriptor.getMaxVersions());
-    hFamily.setCompressionType(Compression.Algorithm.valueOf(descriptor.getCompressionType().name()));
-    hFamily.setBloomFilterType(org.apache.hadoop.hbase.regionserver.BloomType.valueOf(
-      descriptor.getBloomType().name()));
-    for (Map.Entry<String, String> property : descriptor.getProperties().entrySet()) {
-      hFamily.setValue(property.getKey(), property.getValue());
-    }
-    return hFamily;
-  }
-
+  @Override
   public HTableDescriptor getHTableDescriptor(TableDescriptor descriptor) {
-    TableName tableName = TableName.valueOf(descriptor.getNamespace(), descriptor.getName());
-    HTableDescriptor htd = new HTableDescriptor(tableName);
-    for (Map.Entry<String, ColumnFamilyDescriptor> family : descriptor.getFamilies().entrySet()) {
-      htd.addFamily(getHColumnDesciptor(family.getValue()));
-    }
-
-    for (Map.Entry<String, CoprocessorDescriptor> coprocessor : descriptor.getCoprocessors().entrySet()) {
-      CoprocessorDescriptor cpd = coprocessor.getValue();
-      try {
-        htd.addCoprocessor(cpd.getClassName(), new Path(cpd.getPath()), cpd.getPriority(), cpd.getProperties());
-      } catch (IOException e) {
-        LOG.error("Error adding coprocessor.", e);
-      }
-    }
-
-    for (Map.Entry<String, String> property : descriptor.getProperties().entrySet()) {
-      // TODO: should not add coprocessor related properties since those were already be added
-      // using addCoprocessor call.
-      htd.setValue(property.getKey(), property.getValue());
-    }
-    return htd;
+    return HBase96TableDescriptorUtil.getHTableDescriptor(descriptor);
   }
 
+  @Override
   public TableDescriptor getTableDescriptor(HTableDescriptor descriptor) {
```

Comment:
if it's not needed, you can remove that method from the util class and move it into the corresponding HBaseTableUtil class